packages/dev/core/src/Audio/v2/abstractAudioListener.ts

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable babylonjs/available */
+/* eslint-disable jsdoc/require-jsdoc */
+
+import { AbstractAudioNode, AudioNodeType } from "./abstractAudioNode";
+import { Vector3 } from "../../Maths/math.vector";
+import type { AbstractMesh } from "../../Meshes";
+import type { Nullable } from "../../types";
+
+export abstract class AbstractAudioListener extends AbstractAudioNode {
+    public constructor(parent: AbstractAudioNode) {
+        super(parent.engine, AudioNodeType.InputOutput, parent);
+    }
+
+    private _position = Vector3.Zero();
+    private _forward = Vector3.Forward();
+    private _up = Vector3.Up();
```

Comment:
This is an unused class I forgot to remove. I decided listeners don't need to be audio nodes since they only need to provide parameters to audio positioner nodes. The `SpatialAudioListener` class replaces this `AbstractAudioListener` node.

Reply:
Was part of the general note i had regarding getters/setters, which are slightly slower (due to the function call). Avoid them whenever possible and use the if there are some side-effects after setting something.

Reply:
Is it considered a back-compat break if we want to change a property to use getters/setters later? I'm thinking it's not a back-compat break since it looks the same in the API. Is that right?